bash/scripts/milestone_watcher.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+set -eo pipefail
+
+main() {
+  repoName="workflow"
+
+  milestone=$(retrieve-github-milestone "${repoName}")
+  currentVersion="${WORKFLOW_VERSION:-$(cat "$(get-workflow-version-filepath)")}"
+
+  if [ "${milestone}" == "" ]; then
+    echo "No milestone found from the deis/${repoName} repo!"
+  elif [ "${currentVersion}" != "${milestone}" ]; then
+    echo "Bumping current Workflow version from '${currentVersion}' to '${milestone}'..."
+    bump-workflow-version "${milestone}"
+  else
+    echo "Current Workflow version '${currentVersion}' already matches '${milestone}'."
+  fi
+}
+
+retrieve-github-milestone() {
+  repoName="${1}"
+
+  curl \
+  -sSL \
+  "https://api.github.com/repos/deis/${repoName}/milestones" \
+  | docker run -i --rm kamermans/jq '.[].title' | head -n1
```

Comment:
Note: this will pull the milestone verbatim from GitHub.  For example, the current release/milestone is precisely `v2.2` (whereas we had been in the habit of using the full-length semver for our releases, in this case, `v2.2.0`.)  We need to agree on what version will be the standard -- it will be easiest if the GitHub milestone is considered the exact string representation of the next release.